doc/reference/modules/persistent_classes.xml

```diff
@@ -350,8 +372,23 @@ using(ITransaction tx = s.BeginTransaction())
         .List<IDictionary<string, object>>();
     ...
 }]]></programlisting>
+
+        <programlisting><![CDATA[using System.Linq.Dynamic.Core;
+
+...
+
+using(ISession s = OpenSession())
+using(ITransaction tx = s.BeginTransaction())
+{
+    var customers = s
+        .Query<dynamic>("Customer")
```

Comment:
`Query<dynamic>` does not make sense as it does not support filtering/ordering/projecting by expression.

Reply:
It makes sens for retrieving the resulting objects as dynamics. Of course for querying you have to use dynamic expressions too. `Where("query fragment")`, `OrderBy("SomePropertyName")`, ...

Reply:
Anyway, again, I do not really see any practical difference between querying a dynamic entity and querying a dynamic component. Both require the use of dynamic Linq instead of expressions, both are "typed" `dynamic`.